src/functionalTest/resources/scenarios/RIA-7939-internal-detained-non-ada-case-listed.json

```diff
@@ -1,11 +1,11 @@
 {
-  "description": "RIA-7558 Internal Detained non ADA case listed direction notification",
+  "description": "RIA-7939 Internal Detained non ADA case listed direction notification",
```

Comment:
Don't remove previous reference. Add your along with the previous one. like RIA-7558 RIA7939 Internal........